.gitignore

```diff
@@ -4,3 +4,5 @@ node_modules
 .vscode-test/
 *.vsix
 .DS_Store
+package-lock.json
+pnpm*
```

Comment:
What does this pnpm do?

Reply:
Is another package manager, much less popular. If you use it, it creates a few files that start with pnpm hence the wildcard.